storage/cache/subtree_cache.go

```diff
@@ -197,36 +195,26 @@ func (s *SubtreeCache) cacheSubtree(t *storagepb.SubtreeProto) error {
 
 // GetNodes returns the requested nodes, calling the getSubtrees function if
 // they are not already cached.
-func (s *SubtreeCache) GetNodes(nodeIDs []compact.NodeID, getSubtrees GetSubtreesFunc) ([]tree.Node, error) {
-	// TODO(pavelkalinnikov): Use compact.NodeID directly, don't convert.
-	ids := make([]tree.NodeID, len(nodeIDs))
-	for i, nID := range nodeIDs {
-		id, err := tree.NewNodeIDForTreeCoords(int64(nID.Level), int64(nID.Index), maxSupportedTreeDepth)
-		if err != nil {
-			return nil, fmt.Errorf("failed to create nodeID %v: %v", nID, err)
-		}
-		ids[i] = id
-	}
-
+func (s *SubtreeCache) GetNodes(ids []compact.NodeID, getSubtrees GetSubtreesFunc) ([]tree.Node, error) {
 	glog.V(2).Infof("cache: GetNodes(count=%d)", len(ids))
 	if glog.V(3) {
-		for _, n := range ids {
-			glog.Infof("  cache: GetNodes(path=%x, prefixLen=%d)", n.Path, n.PrefixLenBits)
+		for _, id := range ids {
```

Comment:
Might be time to remove logging the IDs. Not sure we'll ever use it.

Reply:
Agree. Doing this separately in #2405, and will rebase this PR.